service/src/main/resources/application.yaml

```diff
@@ -146,7 +146,7 @@ scheduler:
       cronGroup: "NIGHTLY_CRON"
       description: 'Finds upcoming hearings'
       jobClass: 'uk.gov.hmcts.reform.fpl.jobs.UpcomingHearingsFinder'
-      cron: ${UPCOMING_HEARINGS_CRON:0 0 2 ? * MON-FRI *}
+      cron: ${UPCOMING_HEARINGS_CRON:0 0 2 ? * * *}
```

Comment:
Not needed as job runs `!calendarService.isWorkingDay(baseDate)` to determine wether to run or not.

Reply:
should be ok 👍